test/test_raid_creation.py

```diff
@@ -1,9 +1,14 @@
 import unittest
 
-from mod_bot import main
+from mod_bot.main import clean_whitespaces, clean_at_sign
 
 
-class TestGymNameFinder(unittest.TestCase):
+class TestRaidCreation(unittest.TestCase):
 
-    def test_load_gym(self):
-        self.assertEqual(len(main.load_gyms(1, "./test")), 11)
\ No newline at end of file
+    def test_clean_spaces(self):
+        words = ["!raid", " ", "egg5", "arena", "@20h00"]
+        self.assertEqual(clean_whitespaces(words), ["!raid", "egg5", "arena", "@20h00"])
+
+    def test_clean_at_sign(self):
+        words = ["!raid", " ", "egg5", "arena", "@", "20h00"]
```

Comment:
Shouldn't `!raid_egg5_arena_@_20h00` (_ = whitespace) be read as `!raid_egg5_arena_@20h00` (raid starting at 20:00) and not `!raid_egg5_arena_20h00` (raid ending at 20:00) ?
(See also issue #9 ?)

Reply:
Alternatively is the objective of the PR to completely drop the @ ?

Reply:
true, we should discuss that. IMO, @20h00 and 20h00 should give the same result, but I don't know if some people use the second alternative in the current state

Reply:
Most people (at least on the discord I manage) still use the @ form, but it would be easier to drop it imo

Reply:
maybe we can ask them? At least we would know for sure

Reply:
Keep the @, provide a command for time remaining